.gitignore

```diff
@@ -133,3 +133,5 @@ test/mock_filesystem
 .idea
 .pdm.toml
 bma_test_program.py
+/test/output/
```

Comment:
/test shouldn't exist at all, we should remove it both here and in the folder structure

Reply:
We don't want to track the code for the tests?

Reply:
/test should be tracked
any output made in test should be saved in temp files, rather than being built up and gitignored
although maybe its better to have the output folder existing and ignored, for local testing

Reply:
Ok, because that's exactly what I'm doing in my branch right now